unison-src/transcripts/update-on-conflict.output.md

```diff
@@ -75,21 +72,27 @@ x = 1 + 2
   do an `add` or `update`, here's how your codebase would
   change:
   
-    ⍟ These new definitions are ok to `add`:
+    ⍟ These names already exist. You can `update` them to your
+      new definition:
     
       x : Nat
 
 ```
-Update fails on conflicted `x`:
-
 ```ucm
 .merged> update
 
-  x These definitions failed:
+  ⍟ I've updated these names to your new definition:
```

Comment:
Same here, would be cool to say "... resolving a conflict" or something here, but I understand that it might be tricky to do that now.